cypress/support/elements/common/SortedWork.js

```diff
@@ -65,6 +75,12 @@ class SortedWork {
   checkDocumentNotInGroup(groupName, doc) {
     this.getSortWorkGroup(groupName).find(".list .list-item .footer .info").should("not.contain", doc);
   }
+  checkSimpleDocumentInGroup(groupName, doc) {
+    this.getSortWorkGroup(groupName).find('[data-testid="section-document-list"] [data-test="simple-document-item"]').should("have.attr", "title", doc);
+  }
+  checkSimpleDocumentInSubgroup(groupName, subgroupName, doc) {
+    this.getSortWorkSubgroup(groupName, subgroupName).find('[data-test="simple-document-item"]').should("have.attr", "title", doc);
```

Comment:
Note this element has `data-test` attribute, rather than the more standard `data-testid`.  Probably a typo? But unrelated to this PR.